app/config/set_parameters.php

```diff
@@ -57,10 +57,13 @@
 
     $driver = 'array';
     $cacheType = [
-        'CacheMemcache' => ['memcache'],
         'CacheMemcached' => ['memcached'],
-        'CacheApc' => ['apcu', 'apc'],
-        'CacheXcache' => ['xcache'],
+        'CacheApc' => ['apcu'],
+    ];
+    $adapters = [
```

Comment:
Could you explain why Apc and Memcache are no longer available? (in the description of the PR)